pinot-common/src/main/java/org/apache/pinot/common/assignment/InstancePartitionsUtils.java

```diff
@@ -161,6 +161,24 @@ public static InstancePartitions computeDefaultInstancePartitionsForTag(HelixMan
     return instancePartitions;
   }
 
+  public static InstancePartitions computeDefaultInstancePartitionsForTag(HelixManager helixManager,
```

Comment:
We can replace the old method. I checked all the usage, and it is safe to use the new method in `TierConfigUtils`.

@klsince `TierConfigUtils.getTieredInstancePartitionsForSegment()` shouldn't always use default instance partitions, it should use the tier instance partitions if exists. Can you please take a look?

Reply:
Done!